webrtc-stats.html

```diff
@@ -1913,9 +1922,9 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only [= map/exist =]s if the {{RTCRtpTransceiver}} owning this stream has a
-                  {{RTCRtpTransceiver/mid}} value that is not null. this is that
-                  value.
+                  If the {{RTCRtpTransceiver}} owning this stream has a
```

Comment:
Taken from https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-mid
I wonder if inbound-rtp and outbound-rtp need to take care of the edge case where mid was offered but not negotiated but that seems orthogonal

Reply:
I'd update both of them to say "otherwise this member does not [= map/exist =]." so that they're consistent with each other but we also make use of the "map/exist" reference used everywhere else

Reply:
But the edge case you're describing seems orthogonal and not sure it's needed since this is just piggybacking on whatever the transceiver is doing

Reply:
I think we need to distinguish between "exist" and "present"

Reply:
But the [implementation](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/peerconnection/rtc_stats_report.cc;l=137;drc=e5f8ff312fe542f39c7d34b4440f18d1500b15f9) does not make such distinction. It either is there and has a string value or it is not there. We don't support null or even undefined values. If a value on a dictionary is not set, it does not [= map/exist =].

Reply:
Yeah. But I think we should only use exist for things that will never exist, not things that will not be set on some condition

Reply:
But it will never exist for that object. Every getStats call returns a new object, so we only have objects where it always existed and objects where it never existed. The continuation over time is an illusion.

Reply:
Not a hill to die on, i'm happy using the previous text as long as it is done in both locations